lib/preprocessor.cpp

```diff
@@ -292,13 +292,13 @@ static std::string readcondition(const simplecpp::Token *iftok, const std::set<s
         if (sameline(iftok,dtok) && dtok->name && defined.find(dtok->str()) == defined.end() && undefined.find(dtok->str()) == undefined.end())
             configset.insert(dtok->str());
     }
-    std::string cfg;
+    std::string c;
```

Comment:
this spontanously feels unfortunate. c is not a good name. however it is only used in a very small scope so I could accept this. Is it possible to write a better name?

Reply:
First I had `cfgStr`, which I thought was too long. Changed it back.